app/templates/main/pagination.html

```diff
@@ -4,7 +4,7 @@
          aria-label="Pagination">
         {% if results.has_prev == True %}
             <div class="govuk-pagination__prev">
-                <a class="govuk-link govuk-pagination__link"
+                <a class="govuk-link govuk-link__no-visited-color govuk-pagination__link"
```

Comment:
Could you create a tech debt ticket for the backlog for removing those legacy classes in the html and fixing the tests

Reply:
@anthonyhashemi created a ticket for this https://national-archives.atlassian.net/browse/AYR-1055